Resources/Prototypes/_CD/Traits/traits.yml

```
@@ -0,0 +1,12 @@
+﻿- type: trait
+  category: Quirks
+  id: Synthetic
+  name: trait-synth-name
+  description: trait-synth-desc
+  blacklist:
+    components:
+      - Silicon # The only component I could find that is exclusive to IPCs
+      - BorgChassis
+  excludedSpecies: [IPC]
```

put that in the same format as all other lists
```suggestion
  excludedSpecies:
  - IPC
```

---

this isnt even needed because of the blacklist